### PR TITLE
Enable SE-0353 By Default

### PR DESCRIPTION
**Pull request by CodaFi**
[See acceptance](https://forums.swift.org/t/se-0353-constrained-existential-types/56853/20) on Swift Evolution

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci test

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,30 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 ## Swift 5.7
 
+* [SE-0353][]:
```
Comment (hborla) [inline CHANGELOG.md:8]: Slava added a brief description of SE-0353 in https://github.com/apple/swift/pull/59028; could you move this new description to the existing bullet point?

**Review by hborla — APPROVED** [review]
🎉

File: CHANGELOG.md (hunk 1 above)
Comment (CodaFi) [reply CHANGELOG.md]: Good catch, updated.

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Pull request merged**